.github/workflows/test-mlperf-inference-abtf-poc.yml

```diff
@@ -34,6 +35,7 @@ jobs:
           python-version: "3.8"
         include:
           - os: windows-latest
+            docker: ""
```

Comment:
cant we install docker on windows non interactively?

Reply:
@arjunsuresh , there is command line installation method for windows in official docker [website](https://docs.docker.com/desktop/install/windows-install/). Mybe with some workaround its possible?

Reply:
It works but only for Windows docker images which CM does not support currently.